### PR TITLE
[New] Updated Label screen (adapted from Tyleryandt18's Unified diary screen)

### PR DESCRIPTION
**Pull request by sebastianbarry**
Tyler's old PR: https://github.com/e-mission/e-mission-phone/pull/870

This is the continuation of the Unified diary screen changes. View the previous PR to see last changes made.

### Next steps:

_**I DID NOT add the new label screen directive in the old diary screen in the interest of time because it was taking me a while to figure out.** Also, if we are planning on eventually getting rid of the old Diary screen and incorporating it into this new improved Label (with the Diary screen built in) screen, then it may be a waste of time trying to figure this out._

With this and my previous Commits, we should be good to ship this as-is for testing. Then...

### Next steps

- Incorporate the trip guessing percentages (car percentage, bike percentage, walk percentage) in the `infinite-scroll-trip-item` directive
- Incorporate the date-picker functionality from the `/root/main/diary` into the `/root/main/inf_scroll` screen
- Remove the Diary tab from `main.html`
- Change the Infinite scroll tab's translate name from `main-inf-scroll.tab` to `main-diary`, and the icon from `icon="ion-checkmark-round"` to `icon="ion-map"`

### CODE REVIEW

**Comment by sebastianbarry** [comment]
From the old PR containing Tyler's code review changes (https://github.com/e-mission/e-mission-phone/pull/870):

> **Here is my PR with the current response to Tyler's code review: https://github.com/e-mission/e-mission-phone/pull/883**
> 
> There are still a handfull of things to go over; any conversation that doesn't end in _**Confirmed this change as complete**_ still needs to be addressed.

**Comment by sebastianbarry** [comment]
I resolved another display issue:
Before, the top of the label screen had text which was overwriting it's div, and going into the rest of the page:
Screen Shot 2022-09-26 at 10 25 09 AM

I compared it to our current label screen on the `main` branch:
Screen Shot 2022-09-26 at 10 51 41 AM

I then found out the checkbox button was using the wrong style, so I corrected the style and it is now displaying properly again:
Screen Shot 2022-09-26 at 10 55 21 AM

**Comment by sebastianbarry** [comment]
@shankari can you review my code for the last 3 commits?

- https://github.com/e-mission/e-mission-phone/pull/883/commits/41d9702a964cd824802875d64c567dd8202ebf7f
- https://github.com/e-mission/e-mission-phone/pull/883/commits/a9d425e403fbda2c5c276992d6a23ceda673eac7
- https://github.com/e-mission/e-mission-phone/pull/883/commits/5830f953a736295d37f99f3a0bb722838e99d3e6

**Comment by shankari** [comment]
We've spent multiple review sessions on this, so I'm going to merge this and push out a new release to Android + TestFlight for testing.

**Review by shankari — APPROVED** [review]

File: www/templates/survey/multilabel/multi-label-ui.html [hunk 1]
```diff
@@ -1,28 +1,27 @@
-<!-- Label -->
- <div class="row" style="margin-top: 0px">
-  <div ng-repeat="input in userInputDetails" class={{input.width}} style="text-align: center;font-size: 14px;font-weight: 600;" ng-attr-id="{{ 'userinputlabel' + input.name }}" translate>
```
Comment (shankari) [inline www/templates/survey/multilabel/multi-label-ui.html:3]: The `ng-attr-id` from this is missing.

**Comment by shankari** [comment]
@sebastianbarry I found another issue while merging.
The modified `www/templates/survey/multilabel/multi-label-ui.html` does not have the `ng-attr-id`. Are we actually using that id anywhere? If not, we are right to remove it.

File: www/templates/survey/multilabel/multi-label-ui.html (hunk 1 above)
Comment (shankari) [inline www/templates/survey/multilabel/multi-label-ui.html:3]: the `ng-attr-id` attributes from all of the entries are missing.

**Pull request merged**